fix-schemas.js

```
@@ -0,0 +1,67 @@
+const fs = require('fs');
```

ESLint

[error] A `require()` style import is forbidden.

(`@typescript-eslint/no-require-imports`)

---

[error] Replace `'fs'` with `"fs"`

(prettier/prettier)